### ISSUE TITLE
backgroundColor preference not supported

### ISSUE THREAD

**Issue by Delagen** [infil00p/cordova-crosswalk-engine#26]
Cordova build does not start even with master 1.4.0-dev version of plugin not in embedded nor in shared mode

**Comment by fujunwei** [comment]
Could you please describe your steps?

**Comment by Delagen** [comment]
```
cordova add plugin cordova-plugin-crosswalk-webview
```
config
```xml
<widget>
<preference name="xwalkVersion" value="xwalk_core_library:14+"/>
<preference name="xwalkMode" value="embedded"/>
<preference name="xwalkCommandLine" value="--disable-pull-to-refresh-effect"/>
<preference name="xwalkMultipleApk" value="true"/>
</widget>
```
cordova 5.3.1
cordova-android 4.1.1
cordova build android

Make 2 apks: both installed successfully but when run get "unspecified error" alert.
with version 1.2.0 all works fine with latest 14 version and some fixes in .java file for CookieManager changing import namespace

**Comment by fujunwei** [comment]
It can't be reproduced, could you please share your apk to me? what's the version of your android devices?

**Comment by Delagen** [comment]
Nexus 5 Android 5.1.1
Huawei MediaPad Android 4.2.2

https://yadi.sk/d/34HIHOQ1j7jV5

**Comment by Delagen** [comment]
Found issue
```xml
<preference name="BackgroundColor" value="0xff313438"  /> 
```
is not compatible
So UnsupportedOperation exception on setBackgroundColor operation

**Comment by fujunwei** [comment]
Cool. 
Do you want set background color?

**Comment by Delagen** [comment]
No. I simply commented this line in config.xml
So I suppose to add this bug to documentation at least.

**Comment by dpogue** [comment]
BackgroundColor is a core Cordova preference, so this should be supported. `setBackgroundColor()` is a method declared in the Android View class, it's expected that calling it on a subclass (XWalkWebView) won't crash.

**Comment by dotnetwise** [comment]
still an issue in crosswalk 15?

**Comment by dpogue** [comment]
This is fixed with 1.4.0
